README.md

```diff
@@ -24,3 +24,27 @@ logger.info({
   field2: 3,
 })
 ```
+
+### Logging common fields
+
+This allows adding common fields to all log messages. It can be useful, for example, to add the current user to all log messages.
```

Comment:
Dices this, pero no hay ningún this de nada al que referirse.

```suggestion
Create a custom logger to group fields common to all log messages. It can be useful for example to add the current user inside all messages.
```